pkgs/by-name/et/ethercat/kernel.nix

```diff
@@ -0,0 +1,175 @@
+{
+  callPackage,
+  autoreconfHook,
+  pkg-config,
+  stdenv,
+  lib,
+  kernel,
+  ethercat,
+}:
+let
+  common = callPackage ./common.nix { };
+
+  deviceKernelCompatabilities = {
```

Comment:
Not sure what device kernel compatiblities means, but i don't see "generic" here, is it implied that it's compatible with all kernels?

Reply:
Exactly, the `generic` and `ccat` drivers are supported irrespective of the Kernel version.
They are activated by default here:

https://github.com/NixOS/nixpkgs/pull/302959/files#diff-b399f63d866c72ee8718c3af74adc6f25eff65ef6ffa93db3f77fd01974ee85eR40

Reply:
great!, thanks for the clarification